jasmin_services/templates/jasmin_services/service_details.html

```diff
@@ -258,6 +258,44 @@ <h3 class="card-title">Current Requests</h3>
                             {% endfor %}
                         </table>
                     </div>
+                    
+                {% endif %}
+                {% if grants %}
+                    <div class="card">
+                        <div class="card-header">
+                            <h3 class="card-title">Managers & Deputies</h3>
+                        </div>
+                        <table class="table table-hover access-table">
+                            <thead>
+                                <tr>
+                                    <th></th>
+                                    <th>Role</th>
+                                    <th>Name</th>
+                                    <th></th>
+                                </tr>
+                            </thead>
+                            {% for manager in managers%}
+                                <tbody>
+                                    <tr class="active">
+                                        <td class="text-muted" style="width: 1%;"><i></i></td>
+                                    <td><code>{{"MANAGER"}}</code></td>
```

Comment:
I don't think manager needs to be a jinja2 template. You can replace `{{"MANAGER"}}` with `MANAGER` because `{{"MANAGER"}}` just evaluates to the string `MANAGER` anyway.